### PR TITLE
TL Files for Translation

### PR DESCRIPTION
**Pull request by severinolorillajr**
Initial Files that needs to be translated to Filipino / Tagalog.

### CODE REVIEW

File: tl/console-and-shells/i18n-shell.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
+I18N Shell
+##########
+
+<<<<<<< HEAD
+The i18n features of CakePHP use `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
+as their translation source. PO files integrate with commonly used translation tools
+like `Poedit <http://www.poedit.net/>`_.
+=======
```
Comment (markstory) [inline tl/console-and-shells/i18n-shell.rst]: You have merge conflict markers her ad in a few other files. Those will need to be cleaned up.

**Comment by severinolorillajr** [comment]
@markstory Apologies... I have already made the conflict.
I didn't notice that I was using the master branch.
where should i make a PR at master or 3.0 branch?

**Comment by markstory** [comment]
Master branch is for the 2.x docs, while the 3.0 branch is for 3.x. Which version were ypu planning on translating documentation for?

**Comment by severinolorillajr** [comment]
I can do both. Which one is a priority?
Maybe I should start with 3.x?

**Comment by markstory** [comment]
2.x is close to end of life, and will only be getting bug and security fixes going forward. You may want to focus your efforts on 3.x to start with.

Github does not make it possible for you to have your own branch in this repository. However, you can create your own fork, build up your changes there and submit pull requests as you finish files, or on a schedule that works for you.

**Comment by severinolorillajr** [comment]
This is noted.. closing this..
I'll create a new PR to 3.0 branch.

**Pull request closed without merging**